docker-entrypoint.sh

```diff
@@ -1,6 +1,22 @@
 #!/bin/bash
 set -e
 
+DEFAULT_VHOST=${RABBITMQ_DEFAULT_VHOST:-"/"}
+DEFAULT_USER=${RABBITMQ_DEFAULT_USER:-"guest"}
+DEFAULT_PASSWORD=${RABBITMQ_DEFAULT_PASSWORD:-"guest"}
```

Comment:
That's awesome, I'm not much of a bash guy, and you just taught me something new. I like your approach way better.